marlowe-dashboard-client/src/Play/View.purs

```diff
@@ -212,14 +212,15 @@ renderFooter =
 ------------------------------------------------------------
 dashboardLinks :: forall p. Warn (Text "We need to add the dashboard links.") => Array (HTML p Action)
 dashboardLinks =
-  [ link "Market" ""
-  , link "Docs" ""
-  , link "Support" ""
+  -- FIXME: Add link to Docs
+  [ link "Docs" ""
+  {- disabled for phase 1, link "Market" ""
+  , link "Support" "" -}
   ]
 
 iohkLinks :: forall p. Warn (Text "We need to add the IOHK links.") => Array (HTML p Action)
 iohkLinks =
-  [ link "marlowe.io" ""
+  [ link "marlowe.finance" "https://marlowe.finance"
```

Comment:
Aha, _this_ warning we can get rid of. 😄 

But `marlowe-finance.io` instead of `marlowe.finance`, I think.

Reply:
done